src/icon-manifest/languages.ts

```diff
@@ -200,6 +200,8 @@ export const languages: ILanguageCollection = {
   viml: { ids: 'viml', defaultExtension: 'vim' },
   volt: { ids: 'volt', defaultExtension: 'volt' },
   vue: { ids: 'vue', defaultExtension: 'vue' },
+  wasm: { ids: 'wasm', defaultExtension: 'wasm' },
```

Comment:
Since both share the same icon it would be to unify the `ids`, like `wasm: { ids: ['wasm', 'wat'], defaultExtension: 'wasm' },`